schemas/dublin-core/src/main/plugin/dublin-core/index-fields/index.xsl

```diff
@@ -153,10 +192,15 @@
 
       <xsl:variable name="tags"
                     select="dc:subject[. != '']"/>
+
+      <tagNumber>
+        <xsl:value-of select="count($tags)"/>
+      </tagNumber>
+
       <xsl:if test="count($tags) > 0">
         <tag type="object">[
           <xsl:for-each select="$tags">
-            {"default": <xsl:value-of select="concat($doubleQuote, gn-fn-index:json-escape(.), $doubleQuote)"/>}
+            {"default": <xsl:value-of select="concat($doubleQuote, replace(., '&quot;', '\\&quot;'), $doubleQuote)"/>}
```

Comment:
Why not using `gn-fn-index:json-escape` ?

Reply:
Sorry, that was my mistake.

Reply:
Sorry, that was my mistake.